components/net/http_loader.rs

```diff
@@ -659,6 +658,13 @@ pub fn http_redirect_fetch(
     };
 
     // Step 1 of https://w3c.github.io/resource-timing/#dom-performanceresourcetiming-fetchstart
+    // TODO: check origin and timing allow check
+    context
+        .timing
+        .lock()
+        .unwrap()
+        .set_attribute(ResourceAttribute::RedirectStart);
```

Comment:
Given this build error:
```
error[E0308]: mismatched types
   --> components/net/http_loader.rs:666:24
    |
666 |         .set_attribute(ResourceAttribute::RedirectStart);
    |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected enum `net_traits::ResourceAttribute`, found fn item
    |
    = note: expected type `net_traits::ResourceAttribute`
               found type `fn(net_traits::RedirectStartValue) -> net_traits::ResourceAttribute {net_traits::ResourceAttribute::RedirectStart}`
error: aborting due to previous error
```
This should be using `ResourceAttribute::RedirectStart(RedirectStartValue::FetchStart)`.